pkg/daemon/ceph/client/upgrade.go

```diff
@@ -70,10 +123,10 @@ func GetCephMonVersion(context *clusterd.Context) (*cephver.CephVersion, error)
 }
 
 // GetCephVersions reports the Ceph version of each daemon in the cluster
-func GetCephVersions(context *clusterd.Context) (*CephDaemonsVersions, error) {
-	output, err := getCephVersionsString(context)
+func GetCephVersions(context *clusterd.Context, clusterName string) (*CephDaemonsVersions, error) {
```

Comment:
Could we rename this to `GetAllCephDaemonVersions` to clarify that this gets versions for all daemons? My eyes didn't pick up on the `s` at the end of `Versions`.

Reply:
bump